ui/lib/kubernetes/addon/routes/configure.js

```diff
@@ -23,7 +23,7 @@ export default class KubernetesConfigureRoute extends Route {
     controller.breadcrumbs = [
       { label: 'Secrets', route: 'secrets', linkExternal: true },
       { label: resolvedModel.backend, route: 'overview' },
```

Comment:
This one should have `model: resolvedModel.backend` right? To fix the stable identifier issue we've seen pop up in tests?

Reply:
Good catch on these. I'll update.